### PR TITLE
New project - PXD012131

### PR DESCRIPTION
**Pull request by deeptijk**

### CODE REVIEW

**Comment by deeptijk** [comment]
@ypriverol .. I will update you once there are more projects annotated. I have stated with few.

File: annotated-projects/PXD012131/sdrf.tsv [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+Sample 1	run 107	homo sapiens	brain	Visual Cortex	not applicable	80Y	not applicable	female	Normal	adult	1	107	PXD012131	20171025_JMM_VC146_11.raw	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2020/01/20171025_JMM_VC146_11.raw	1	107	not applicable	1	NT=label free sample	0	not available	05:00	NA	No tangle	AC=MS:1002732;NT=Orbitrap Fusion Lumos	20 ppm	0.5 Da	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Acetyl;AC=UNIMOD:1;PP=Protein N-Term;MT=Variable	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	Normal	female
+Sample 1	run 108	homo sapiens	brain	Visual Cortex	not applicable	80Y	not applicable	female	Normal	adult	1	108	PXD012131	20171025_JMM_VC146_12.raw	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2020/01/20171025_JMM_VC146_12.raw	1	108	not applicable	1	NT=label free sample	0	not available	05:00	NA	No tangle	AC=MS:1002732;NT=Orbitrap Fusion Lumos	20 ppm	0.5 Da	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Acetyl;AC=UNIMOD:1;PP=Protein N-Term;MT=Variable	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	Normal	female
```
Comment (ypriverol) [inline annotated-projects/PXD012131/sdrf.tsv:109]: 108 fractions? Is that correct?

Comment (ypriverol) [inline annotated-projects/PXD012131/sdrf.tsv:109]: No tangle or No tangles?

**Review by ypriverol — CHANGES_REQUESTED** [review]

File: annotated-projects/PXD012131/sdrf.tsv (hunk 1 above)
Comment (deeptijk) [reply annotated-projects/PXD012131/sdrf.tsv:109]: Yes

Comment (deeptijk) [reply annotated-projects/PXD012131/sdrf.tsv]: Uploaded the new sdrf. Corrected the word

**Pull request merged**